Library/Formula/jython.rb

```diff
@@ -1,7 +1,7 @@
 class Jython < Formula
   homepage "http://www.jython.org"
-  url "https://search.maven.org/remotecontent?filepath=org/python/jython-installer/2.5.3/jython-installer-2.5.3.jar"
-  sha256 "05405966cdfa57abc8e705dd6aab92b8240097ce709fb916c8a0dbcaa491f99e"
+  url "https://search.maven.org/remotecontent?filepath=org/python/jython-installer/2.7.0/jython-installer-2.7.0.jar"
+  sha1 "b08d73b5df7d95e35e7b8dcaf1558025cf24f0f1"
```

Comment:
We still need to use it, you just need to confirm the output of `brew fetch jython` yourself.